From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Kleinschmidt <jkleinsc@electronjs.org>
Date: Mon, 10 Jun 2024 11:18:56 -0500
Subject: chore: remove calls to v8::FunctionCallbackInfo::Holder()

Use This() instead.
Refs: https://chromium-review.googlesource.com/c/v8/v8/+/5604487
https://issues.chromium.org/issues/333672197

This patch can be removed once Node addresses this change.

diff --git a/src/node_internals.h b/src/node_internals.h
index 5dff80ee287256ba40bfa496df6db60a18fbb9d7..a9162b4e2ee48dbebec8dc52b6990369be15fe4a 100644
--- a/src/node_internals.h
+++ b/src/node_internals.h
@@ -67,7 +67,7 @@ template <typename T, int (*F)(const typename T::HandleType*, sockaddr*, int*)>
 void GetSockOrPeerName(const v8::FunctionCallbackInfo<v8::Value>& args) {
   T* wrap;
   ASSIGN_OR_RETURN_UNWRAP(&wrap,
-                          args.Holder(),
+                          args.This(),
                           args.GetReturnValue().Set(UV_EBADF));
   CHECK(args[0]->IsObject());
   sockaddr_storage storage;
